src/modules/scores.js

```diff
@@ -1,6 +1,7 @@
 export default class Scores {
-  constructor(name, score) {
-    this.name = name;
-    this.score = score;
+  constructor(userName, points) {
+    userName = this.userName;
+    // eslint-disable-next-line no-unused-vars
+    points = this.points;
```

Comment:
- [ ] Let's try not to disable Linter checks as they help us write clean and professional code. Disabling Linter checks defies its importance. Please fix the error 👍🏽